go-controller/pkg/metrics/ovs.go

```diff
@@ -569,8 +586,28 @@ func ovsBridgeMetricsUpdate(stopChan <-chan struct{}) {
 				klog.Errorf("Getting interface to port bridge mapping failed: %s", err.Error())
 				continue
 			}
+			for interfaceId, interfaceInfo := range existingInterfaceInfo {
+				if _, ok := interfaceToPortToBridgeMap[interfaceId]; ok {
+					continue
+				}
+				// remove non-existent ovs inteface from existingInterfaceInfo map and its associated metrics.
+				delete(existingInterfaceInfo, interfaceId)
```

Comment:
Why not just call Reset() on all these metrics instead of trying to store the interface info? Its simpler.

For the record, I don't like that we expose all this information for each interface using labels which have unbounded cardinality but for now, we need to stop the memory ballooning for now.

Reply:
Yes Martin, we can just use reset function here, good find !

I too agree with you for disabling ovs interface metrics as i don't see much significance in exposing it into NB via prometheus. Because ovs interface metric labels used here doesn't have direct mapping with pod, so i'm not sure how its going to get used. Looks like interface name label contains pod sandbox id prefix (as per [veth](https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/cni/helper_linux.go#L129) creation) and is that what used now to correlate a metric with relevant pod ?